splite writes:
The netstat(8) man page in sys-apps/net-tools states that the "-i" and
"--interfaces" flags to netstat take an optional argument denoting a 
particular interface when it in fact does not.

http://bugs.gentoo.org/show_bug.cgi?id=29677

--- man/en_US/netstat.8.orig	2003-09-26 10:08:19.000000000 -0500
+++ man/en_US/netstat.8	2003-09-26 10:25:39.000000000 -0500
@@ -44,7 +44,6 @@
 .P
 .B netstat
 .RB { \-\-interfaces | \-i }
-.RI [ iface ]
 .RB [ \-\-all | \-a ]
 .RB [ \-\-extend | \-e  [ \-\-extend | \-e] ]
 .RB [ \-\-verbose | \-v ]
@@ -107,9 +106,8 @@
 Display the kernel routing tables.
 .SS "\-\-groups , \-g"
 Display multicast group membership information for IPv4 and IPv6.
-.SS "\-\-interface=\fIiface \fR, \fB\-i"
-Display a table of all network interfaces, or the specified
-.IR iface ) .
+.SS "\-\-interface , \-i"
+Display a table of all network interfaces.
 .SS "\-\-masquerade , \-M"
 Display a list of masqueraded connections.
 .SS "\-\-statistics , \-s"
